src/tests/unexpected-react-shallow.spec.js

```diff
@@ -1,7 +1,8 @@
 var Unexpected = require('unexpected');
 var UnexpectedReact = require('../unexpected-react');
 
-var React = require('react/addons');
+const React = require('react');
+const TestUtils = require('react-addons-test-utils');
```

Comment:
I would either stick with var or update the whole file to const. Just for consistency